index.bs

```diff
@@ -2289,10 +2297,33 @@ during registration.
     :   {{AuthenticatorAttestationResponse/getTransports()}}
     ::  This operation returns the value of {{AuthenticatorAttestationResponse/[[transports]]}}.
 
+    :   {{AuthenticatorAttestationResponse/getAuthenticatorData()}}
+    ::  This operation returns the [=authenticator data=] contained within {{AuthenticatorAttestationResponse/attestationObject}}. See [[#sctn-public-key-easy]].
+
+    :   {{AuthenticatorAttestationResponse/getPublicKey()}}
+    ::  This operation returns a string that contains the base64-encoded, DER [=SubjectPublicKeyInfo=] of the new credential, or an empty string if this is not available. See [[#sctn-public-key-easy]].
+
     :   <dfn>\[[transports]]</dfn>
     ::  This [=internal slot=] contains a sequence of zero or more unique {{DOMString}}s in lexicographical order. These values are the transports that the [=authenticator=] is believed to support, or an empty sequence if the information is unavailable. The values SHOULD be members of {{AuthenticatorTransport}} but [=[RPS]=] MUST accept unknown values.
 </div>
 
+#### Easily accessing credential data #### {#sctn-public-key-easy}
+
+Every user of the [[#sctn-createCredential|create method]] will need to parse and store the returned public key in order to verify future assertions. However, the public key is in [[!RFC8152]] (COSE) format, inside the [=credentialPublicKey=] member of the [=attestedCredentialData=], inside the [=authenticator data=], inside the [=attestation object=] from {{AuthenticatorAttestationResponse/attestationObject}}. Users that wish to use [=attestation=] MUST do the work of parsing that out because that copy of the public-key is the one that the [=authenticator=] has [signed](#signing-procedure). However, many valid uses of WebAuthn do not need [=attestation=]. For those uses, user agents can do the work of parsing, expose the [=authenticator data=] directly, and translate the public key into a more convenient format.
```

Comment:
We use `[=base64url encoding=]` throughout the spec. Ought we also here in this section?  If not we perhaps ought to note that and explain why.

Editorial polishing suggestion to more align with rest of spec:
```suggestion
Every user of the {{PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)}} method will need to parse and store the returned [=credential public key=] in order to verify future [=authentication assertions=]. However, the [=credential public key=] is in [[!RFC8152]] (COSE) format, inside the [=credentialPublicKey=] member of the [=attestedCredentialData=], inside the [=authenticator data=], inside the [=attestation object=] conveyed by {{AuthenticatorAttestationResponse}}.{{AuthenticatorAttestationResponse/attestationObject}}. [=[RPS]=] wishing to use [=attestation=] are obliged to do the work of parsing the {{AuthenticatorAttestationResponse/attestationObject}} and obtaining the [=credential public key=]  because that public key copy is the one the [=authenticator=] [signed](#signing-procedure). However, many valid WebAuthn use cases do not require [=attestation=]. For those uses, user agents can do the work of parsing, expose the [=authenticator data=] directly, and translate the [=credential public key=] into a more convenient format.
```

Reply:
> We use `[=base64url encoding=]` throughout the spec. Ought we also here in this section?

I'm inclined to agree, it seems good to stick with base64url as much as possible.

Reply:
Actually, on second thought, maybe this should be raw `ArrayBuffer`? See: https://github.com/w3c/webauthn/pull/1395#discussion_r409055892

Reply:
(I'm keeping an eye on this for https://github.com/github/webauthn-json)

While I'm not a fan of all the `ArrayBuffer`s, everything binary that is passed/returned in the browser API is an `ArrayBuffer` and I think by far the least confusing thing would be to stay consistent. It might also save code for application who want to process on the frontend but don't otherwise need a `base64url` decoder.